### ISSUE TITLE
Can't Disable Indication

### ISSUE THREAD

**Issue by GsMoMo** [NordicSemiconductor/Android-BLE-Library#316]
When we try to disable the indication of a characteristic by calling the function "disableIndications" in "BleManagerHandler.java", the function always returns the error code -3 (The null attribute). Therefore, I traced back to the code base and found the function "disableIndications" is as same as "disableNotification". From my point of view, when it tries to get the descriptor it should use the same property type as the function "enableIndications" does as below:

```java
final BluetoothGattDescriptor descriptor = getCccd(characteristic, BluetoothGattCharacteristic.PROPERTY_INDICATE);
```
Could you check this issue? Thanks.

**Comment by philips77** [comment]
You're right. Looks like a bug here:
https://github.com/NordicSemiconductor/Android-BLE-Library/blob/58dde71866bc0e796c396d99aa3cceac5e940f44/ble/src/main/java/no/nordicsemi/android/ble/BleManagerHandler.java#L811
This is called from:
https://github.com/NordicSemiconductor/Android-BLE-Library/blob/58dde71866bc0e796c396d99aa3cceac5e940f44/ble/src/main/java/no/nordicsemi/android/ble/BleManagerHandler.java#L844-L847

**Comment by philips77** [comment]
Thank you for the report! I'll fix it asap.

**Comment by GsMoMo** [comment]
Thanks a lot!

**Comment by philips77** [comment]
Will be released in 2.3.1.
